github-for-jira.sd.yml

```diff
@@ -204,6 +204,22 @@ alarms:
     HighSeverityAlarmWhenTooManyELB5xxErrors: null
     LowSeverityAlarmWhenTooManyBackend5xxErrors: null
     LowSeverityAlarmWhenTooManyELB5xxErrors: null
+  eventsQueueSize:
+    MetricName: ApproximateNumberOfMessagesVisible
+    Namespace: AWS/SQS
+    Description: "Lots of messages in the events queue. Follow the runbook: TBD"
+    # Threshold is 10 because messages generally process fairly quickly but sometimes
+    # we'll have 10 or less that end up waiting on the queue for a short duration
+    Threshold: 10
```

Comment:
We should set it to the value greater than our autoscaling threshold (See the autoscaling rules for calculation).

For example 400 for push, 
20 for Backfill queue
5 for Discovery queue (Because we rarely get messages on it, so 5 pending messages will be a huge amount)